### PR TITLE
[keyboard] jacky_studio/piggy60

### PR DESCRIPTION
**Pull request by lesshonor**
## Description

Adds Jacky's [Piggy60](https://jackylab.com/products/piggy-60-gasket-mount-keyboard) to QMK.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/jacky_studio/piggy60/config.h [hunk 1]
```diff
@@ -0,0 +1,4 @@
+// Copyright 2022 The QMK Community (@qmk)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#define LED_PIN_ON_STATE 0
```
Comment (drashna) [inline keyboards/jacky_studio/piggy60/config.h:4]:
```suggestion
#pragma once

#define LED_PIN_ON_STATE 0
```

File: keyboards/jacky_studio/piggy60/readme.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Piggy60
+
+![piggy60](https://i.imgur.com/QXuKDua.jpg)
```
Comment (drashna) [inline keyboards/jacky_studio/piggy60/readme.md:3]:
```suggestion
![piggy60](https://i.imgur.com/QXuKDual.jpg)
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/jacky_studio/piggy60/config.h [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// Copyright 2022 The QMK Community (@qmk)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define LED_PIN_ON_STATE 0
```
Comment (drashna) [inline keyboards/jacky_studio/piggy60/config.h:6]: Also, just a heads up, you can use "on_state" as true/false under "indicators" in the json file to set this.

**Review by drashna — APPROVED** [review]

File: keyboards/jacky_studio/piggy60/config.h (hunk 3 above)
Comment (drashna) [reply keyboards/jacky_studio/piggy60/config.h:6]: Ah, no worries, then.  

And if it helps, develop is scheduled to be merged into master at the end of the month (in 2 weeks).  Also, qmk.tzarc.io compiles firmware images from develop.

**Review by keyboard-magpie — APPROVED** [review]
Approving in current state, but noting the conversation had regarding indicators

**Pull request merged**